modelling/src/main/java/org/axonframework/modelling/saga/SagaLifecycle.java

```diff
@@ -87,6 +89,15 @@ public static void end() {
         getInstance().doEnd();
     }
 
+    /**
+     * Retrieves all current {@link AssociationValue} that are currently associated with the saga that is being
+     * executed.
+     * @return All current {@link AssociationValue} that are currently associated with the saga
```

Comment:
Small suggestions here.
The second word within an `@link` block will become the actual thing shown, linking to the first entry. That way, we can make things plural, or merge framework class/concepts into the terms used towards users.

```suggestion
     * Retrieves all current {@link AssociationValue AssociationValues} that are currently associated with the saga that is being
     * executed.
     *
     * @return All current {@link AssociationValue} that are currently associated with the saga.
```

Reply:
Awesome, did not know that! I'll read up some more on javadoc